tasks/ext.rake

```diff
@@ -114,4 +107,4 @@ file "#{tmp}/#{taglib}.tar.gz" => [tmp] do |t|
   end
 end
 
-directory tmp
+directory Taglib.tmp
```

Comment:
nit: Let's move this up to the other directories (unless there's a reason it's down here? I forgot).